src/components/media_control/media_control.js

```diff
@@ -139,6 +144,32 @@ class MediaControl extends UIObject {
     this.trigger(Events.MEDIACONTROL_MOUSELEAVE_SEEKBAR, event);
   }
 
+  mousemoveOnVolumeBar(event) {
+    if(this.volumeBarClickDown){
+      this.volume(event);
+    }
+  }
+
+  mousemoveOnVolumeBar(event) {
```

Comment:
Duplicate class method 'mousemoveOnVolumeBar'.<br>'event' is defined but never used.